### ISSUE TITLE
No error message when using the same label in a view

### ISSUE THREAD

**Issue by SterredenBreeijen** [AmpersandTarski/Ampersand#943]
#### Version of ampersand that was used 
3.16.0 master ee22d2045

#### What I expected
Get an error explaining why my `VIEW` was not working.

#### What happened instead
The `VIEW` was compiled, but at runtime it did not function in the way that I expected.

### Steps to reproduce
1. 
2.
3.
4.

#### Context / Source of ampersand script
This does not work: 
~~~
VIEW JsonMessage : JsonMessage
   { "": TXT "{@type:"
   , "type" : hasMessage~;didPairHasType 
   , " ": TXT ", @label:"  --TXT "{php} "
   , "label" : hasMessage~;isOwner~  --"agent"
   , " ":TXT ", @endpoint:"
   , "endpoint" : I --"??"
   , "":TXT ", @status:"
   , "Status" : hasMessage~;statusOfPair  --"status"
   , "":TXT ", @id:"
   , "id": hasMessage~ --"random string"
   , "":TXT "}"
   } ENDVIEW
~~~

This works: 
~~~
VIEW JsonMessage : JsonMessage
   { "1": TXT "{@type:"
   , "type" : hasMessage~;didPairHasType 
   , "2": TXT ", @label:"  --TXT "{php} "
   , "label" : hasMessage~;isOwner~  --"agent"
   , "3":TXT ", @endpoint:"
   , "endpoint" : I --"??"
   , "4":TXT ", @status:"
   , "Status" : hasMessage~;statusOfPair  --"status"
   , "5":TXT ", @id:"
   , "id": hasMessage~ --"random string"
   , "6":TXT "}"
   } ENDVIEW
~~~

**Comment by RieksJ** [comment]
@hanjoosten : the issue is that the erroneous `VIEW` has labels that are the same. In `INTERFACE`s, that is an error (which I believe is actually generated). This should also be the case for `VIEW`s.

**Comment by Michiel-s** [comment]
Duplicate of #1171 and fixed by #1179
